source/javascript/components/elb/list.html

```diff
@@ -0,0 +1,55 @@
+<section class="md-whiteframe-2dp md-padding">
+    <filters filters="vm.filters" on-update="vm.updateFilters" on-reset="vm.resetFilters"></filters>
+
+    <md-table-pagination
+            style="border: 0;"
+            md-on-paginate="vm.updatePath"
+            md-page="vm.params.page"
+            md-page-select
+            md-limit="vm.params.count"
+            md-limit-options="[10, 20, 50]"
```

Comment:
per comments on the collector if we can go with 25,50,100 for the counts would be great.